docs/README.md

```diff
@@ -214,28 +258,28 @@ The mobile map package contents can be accessed after the mobile map package has
 ```xml
 <!-- Mmpk Thumbnail area -->
 <Grid>
-	<Image Name="Thumbnail" VerticalAlignment="Center" HorizontalAlignment="Center" Width="300" Height="100"
-	   Source="{Binding MmpkItem.Thumbnail, Converter={StaticResource RuntimeImageConverter}}"
-	   Grid.Row="0"/>
+  <Image Name="Thumbnail" VerticalAlignment="Center" HorizontalAlignment="Center" Width="300" Height="100"
+     Source="{Binding MmpkItem.Thumbnail, Converter={StaticResource RuntimeImageConverter}}"
+     Grid.Row="0"/>
 </Grid>
 
 <!-- Mmpk name area-->
 <TextBlock Text="{Binding MmpkItem.Title}" Grid.Row="1" Style="{StaticResource WhiteText}"
-		FontSize="27" FontWeight="Bold" VerticalAlignment="Top" TextWrapping="Wrap"
-			   Margin="10,20,10,20" Foreground="#3F51B5"/>
+    FontSize="27" FontWeight="Bold" VerticalAlignment="Top" TextWrapping="Wrap"
+         Margin="10,20,10,20" Foreground="#3F51B5"/>
 
 <!-- Mmpk Description Area -->
 <StackPanel Orientation="Vertical" Margin="10,0,10,0" Grid.Row="2">
-	<TextBlock Text="{Binding MmpkItem.Description}" TextWrapping="Wrap"
-		   Style="{StaticResource GrayText}" Margin="0,20,0,0" />
-	<StackPanel Orientation="Horizontal" Margin="0,20,0,0">
-		<TextBlock Text="Maps: " Style="{StaticResource GrayText}" FontStyle="Italic"/>
-		<TextBlock Text="{Binding MapItems.Count}" Style="{StaticResource GrayText}" FontStyle="Italic"/>
-	</StackPanel>
+  <TextBlock Text="{Binding MmpkItem.Description}" TextWrapping="Wrap"
+       Style="{StaticResource GrayText}" Margin="0,20,0,0" />
```

Comment:
In this case, I think the line should be indented with spaces such that `Style` aligns with `Text` above.